app/models/schema_contract/validation.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module SchemaContract
+  class Validation < ApplicationRecord
+    self.table_name = 'schema_contract_validations'
+
+    attribute :contract_name, :string
+    attribute :user_uuid, :string
+    attribute :response, :jsonb
+    attribute :error_details, :string
+
+    validates :contract_name, presence: true
+    validates :user_uuid, presence: true
+    validates :response, presence: true
+    validates :status, presence: true
+
+    enum status: { initialized: 0, success: 1, schema_errors_found: 2, schema_not_found: 3, error: 4 }
```

Comment:
If you wanted, you could make this simpler and just do
```
enum :status, [ :initialized, :success, :schema_errors_found, :schema_not_found, :error]
```

You can also set a default
```
enum :status, [ :initialized, :success, :schema_errors_found, :schema_not_found, :error], default: :initialized
```

But if you like the more verbose approach because I have no complaints

Reply:
I'll try adding the default, but when I tried writing it as an array the linter reformatted it to the hash.